### PR TITLE
chore: rename `whaction` and `action` to `resource_action` in CLI

### PR DESCRIPTION
**Pull request by JohnVillalovos**
Rename the variables `whaction` and `action` to `resource_action` to
improve code-readability.

### CODE REVIEW

File: gitlab/v4/cli.py [hunk 1]
```diff
@@ -351,7 +355,7 @@ def extend_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         object_group = subparsers.add_parser(arg_name)
 
         object_subparsers = object_group.add_subparsers(
-            title="action", dest="whaction", help="Action to execute."
+            title="resource_action", dest="resource_action", help="Action to execute."
```
Comment (nejch) [inline gitlab/v4/cli.py:358]: The `title` is a user-facing human-readable string for the help so I think we could keep it, e.g. I think this is more consistent with other CLI prints:

```
usage: gitlab project-access-token [-h] {list,create,delete} ...

optional arguments:
  -h, --help            show this help message and exit

action:
  {list,create,delete}  Action to execute.
```

..than the new:
```
usage: gitlab project-access-token [-h] {list,create,delete} ...

optional arguments:
  -h, --help            show this help message and exit

resource_action:
  {list,create,delete}  Action to execute.
```
We could instead maybe extend the help string to say `Action to execute on the resource.`

**Review by nejch — CHANGES_REQUESTED** [review]
Thanks @JohnVillalovos! looks good just a tiny comment about the help string.

File: gitlab/v4/cli.py (hunk 1 above)
Comment (nejch) [inline gitlab/v4/cli.py:358]:
```suggestion
            title="action", dest="resource_action", help="Action to execute."
```

Comment (JohnVillalovos) [reply gitlab/v4/cli.py]: Thanks. Done.

Comment (JohnVillalovos) [reply gitlab/v4/cli.py]: Thanks. Good point. Done.

**Review by nejch — APPROVED** [review]

**Pull request merged**